init-dallas-temperature.h

```diff
@@ -37,7 +38,7 @@ void initDallasSensor()
 
   // find sensor:
   // find sensor addr by index.
-  for (uint8_t i = 0; i < connectedSensors; ++i)
+  for (uint8_t i = 0; i < connectedSensors || i < MAX_DALLAS_TEMPERATURE_SENSORS_COUNT; ++i)
```

Comment:
I guess that here we need AND condition.
Cause number of connected devices might be > MAX_DALLAS_TEMPERATURE_SENSORS. And in this case we`ll get undefined behavior(cause of accessing memory out of array range).

Reply:
Yes
Commited but forgot to push